trace/tracestate.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const (
+	KEY_WITHOUT_VENDOR_FORMAT = `[a-z][_0-9a-z\-\*\/]{0,255}`
```

Comment:
Go convention for consts is camelCase - I think golint will pick this up

Reply:
Sure, I'd be glad to change to camelCase.

I did searched the entire repo to see if there is a convention I could follow, and I'm seeing inconsistent conventions. And golint is not part of CI validation (all I see is gofmt). e.g.

In stats/validation.go, we have
```
const (
	MaxNameLength = 255
)
```

in vendor\git.apache.org\thrift.git\lib\go\thrift\application_exception.go (although this is 3rd party dependency), we have
```
const (
	UNKNOWN_APPLICATION_EXCEPTION  = 0
	UNKNOWN_METHOD                 = 1
	INVALID_MESSAGE_TYPE_EXCEPTION = 2
	WRONG_METHOD_NAME              = 3
	BAD_SEQUENCE_ID                = 4
	MISSING_RESULT                 = 5
	INTERNAL_ERROR                 = 6
	PROTOCOL_ERROR                 = 7
)
```